styles/colours/colourpalette.md

```diff
@@ -11,48 +11,47 @@ The B.C. Government colour palette must be adhered to ensuring all public facing
+![#afafaf](https://github.com/bcgov/design-system/blob/master/styles/colours/images/afafaf.png?raw=true) | #afafaf | 175 175 175 | ???
+![#616265](https://github.com/bcgov/design-system/blob/master/styles/colours/images/616265.png?raw=true) | #616265 | 97 98 191 | ???
+![#8a9198](https://github.com/bcgov/design-system/blob/master/styles/colours/images/8a9198.png?raw=true) | #8a9198 | 138 145 152 | ???
+![#002663](https://github.com/bcgov/design-system/blob/master/styles/colours/images/002663.png?raw=true) | #002663 | 0 38 99 | ???
+![#5091cd](https://github.com/bcgov/design-system/blob/master/styles/colours/images/5091cd.png?raw=true) | #5091cd | 80 145 205 | ???
+![#96c0e6](https://github.com/bcgov/design-system/blob/master/styles/colours/images/96c0e6.png?raw=true) | #96c0e6 | 150 192 230 | ???
+![#f3bd48](https://github.com/bcgov/design-system/blob/master/styles/colours/images/f3bd48.png?raw=true) | #f3bd48 | 243 189 72 | ???
+![#f3cd65](https://github.com/bcgov/design-system/blob/master/styles/colours/images/f3cd65.png?raw=true) | #f3cd65 | 243 203 101 | ???
+![#fade81](https://github.com/bcgov/design-system/blob/master/styles/colours/images/fade81.png?raw=true) | #fade81 | 250 222 129 | ???
+![#486446](https://github.com/bcgov/design-system/blob/master/styles/colours/images/486446.png?raw=true) | #486446 | 72 100 70 | ???
+![#481124](https://github.com/bcgov/design-system/blob/master/styles/colours/images/481124.png?raw=true) | #481124 | 72 33 36 | ???
+![#d8292f](https://github.com/bcgov/design-system/blob/master/styles/colours/images/d8292f.png?raw=true) | #d8292f | 216 41 47 | ???
+![#a12622](https://github.com/bcgov/design-system/blob/master/styles/colours/images/a12622.png?raw=true) | #a12622 | 161 38 34 | ???
 
 ## Rationale
 Our brand colours, BCID Blue and BCID Gold, are from our symbol, the BC Mark. Golden Yellow symbolizes warmth, energy, balance, enthusiasm, vibrancy and vitality. It is also associated with optimism, prosperity, prestige, wisdom, intelligence and security. Dark Blue symbolizes depth, expertise, stability, credibility, knowledge and integrity. It is also associated with health, law, order, logic, and dependability. 
 
 ## Accessibility
 This navigation bar has been built according to WCAG 2.0 AA standards and has been successfully tested for the following:
-1.	Colour Contrast
-*	WCAG 2.0 AAA standards requires normal text (18.66px and under) to have at least 7:1 ratio and large text (18.66 and bolded or 24px) to have at least a 4.5:1 ratio. 
-
 
+### Colour Contrast
+*	WCAG 2.0 AAA standards requires normal text (18.66px and under) to have at least 7:1 ratio and large text (18.66 and bolded or 24px) to have at least a 4.5:1 ratio. 
 
 ## Design with This Component
 Download and use the B.C. government Design System library to help build prototypes and wireframes for your website or application. Prototype make the hand-off from designers to developers easier and faster.
```

Comment:
I just noticed that this "Prototype" text is phrased a little differently than the others. I like it, but noticed a small type: "PrototypeS make the hand-off from designers....: